docs/accessControl.md

```diff
@@ -64,15 +64,14 @@ Every course belongs to an **institution** and by default access is only allowed
-| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Public`    | Access is normally allowed, but is blocked when the student is checked in on PrairieTest, either in a testing center or in a course-managed session.                                                                                                          |
-| `Exam`      | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a testing center or in a course-managed session. The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
-| no mode set | This is equivalent to setting the `mode` to `Public`.                                                                                                                                                                                                         |
+| Mode     | When is access allowed?                                                                                                                                                                                       |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Public` | Access is normally allowed, but is blocked when the student is checked in on PrairieTest, either in a testing center or in a course-managed session. This is the default mode.                                |
+| `Exam`   | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a testing center or in a course-managed session. This mode is used when the `examUuid` is specified. |
 
-In general usage it is best to set `"mode": "Public"` for any homework (assessments that students should do at home or without special access control), and to set both `"mode": "Exam"` and `"examUuid"` for exams with access controlled by PrairieTest. This will make it so that exams are only accessible in a testing center or a controlled course session, and homework is _not_ accessible during exams.
+In general usage it is best to keep the Public mode for any homework (assessments that students should do at home or without special access control), and to use exam mode, either with an explicit `"mode": "Exam"` setting or by setting the `"examUuid"`, for exams with access controlled by PrairieTest. This will make it so that exams are only accessible in a testing center or a controlled course session, and homework is _not_ accessible during exams.
```

Comment:
```suggestion
In general usage it is best to keep the default Public mode for any homework (assessments that students should do at home or without special access control), and to use exam mode, by setting the `"examUuid"`, for exams with access controlled by PrairieTest. This will make it so that exams are only accessible in a testing center or a controlled course session, and homework is _not_ accessible during exams.
```